api/Document.json

```diff
@@ -8901,6 +8901,53 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "text_param_accepts_TrustedHTML": {
+          "__compat": {
+            "description": "`text` parameter accepts `TrustedHTML` type",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/write",
+            "spec_url": "https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-document-write",
+            "tags": [
+              "web-features:dom",
+              "web-features:trusted-types"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "83"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "133",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.security.trusted_types.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": true
```

Comment:
Note, this is set as not experimental and deprecated, because the parent is deprecated (and you can't have both set).